PurchasesCoreSwift/Networking/HTTPRequest.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-public typealias HTTPClientResponseHandler = (Int, [String: Any]?, Error?) -> Void
+public typealias HTTPClientResponseHandler = (_ statusCode: Int, _ response: [AnyHashable: Any]?, _ error: Error?) -> Void
```

Comment:
these typealiases aren't visible in obj-c anyway, right? I kinda feel like we don't need them